examples/crm/src/contacts/ContactInputs.tsx

```diff
@@ -29,7 +38,64 @@ const isLinkedinUrl = (url: string) => {
 };
 
 export const ContactInputs = () => {
+    const theme = useTheme();
+    const isMobile = useMediaQuery(theme.breakpoints.down('sm'));
```

Comment:
**nit-pick:** I would have gone for `md` break point here to detect mobile, otherwise the phone number field is too tight